### PR TITLE
data-seeding function and script

### PR DESCRIPTION
**Pull request by ZsokaM**

### CODE REVIEW

File: seed-data/data.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const weekNum = i
+    for (let y = 0; y <= 6; y++) {
+      const dayNum = y
+      for (let k = 8; k <= 20; k++) {
+        const startHour = k
+        const myClass = {
+          name: classTypes[Math.floor(Math.random() * classTypes.length)],
+          freeSpots: 10,
+          available: 1,
+          year: 2021,
```
Comment (michaelkoek) [inline seed-data/data.ts:34]: Why not get the current year with the `new Data()` class?

File: seed-data/data.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          week: weekNum,
+          day: dayNum,
+          startTime: startHour,
+          teacher: teachers[Math.floor(Math.random() * teachers.length)],
+          duration: 60,
```
Comment (michaelkoek) [inline seed-data/data.ts:39]: Duration of what? Duration in weeks? days? minutes? hours?
`durationInMinutes` would be clearer

File: seed-data/data.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+]
+
+let classArr = []
+
+const generateClasses = () => {
+  for (let i = 35; i <= 41; i++) {
+    const weekNum = i
+    for (let y = 0; y <= 6; y++) {
+      const dayNum = y
+      for (let k = 8; k <= 20; k++) {
```
Comment (michaelkoek) [inline seed-data/data.ts:28]: As you already know what are the start- and end time, why not put these values outside of the function and use it where you need it.

```
let startTime = 8;
let endTime = 20;

for (let k = startTime; k <= endTime; k++) {
```

Actually, I wondering why you need this loop. You know how many hours are in between 8 and 20. it's something that doesn't change

File: seed-data/data.ts (hunk 2 above)
Comment (ZsokaM) [reply seed-data/data.ts:39]: this is just an easy function to generate dummy data so i have weeks worth of classes. duration is the expected field of the schema.

File: seed-data/data.ts (hunk 3 above)
Comment (ZsokaM) [reply seed-data/data.ts:28]: i need a class for every day, every hour, again, this is just to generate dummy data.

**Pull request merged**